snuba/datasets/events.py

```diff
@@ -18,6 +20,16 @@
 from snuba.query.timeseries_extension import TimeSeriesExtension
 from snuba.util import qualified_column
 
+# TODO: This will be a property of the relationship between entity and
```

Comment:
fyi: style, this could be inlined where it is used, but it felt less readable with the three lines comment attached.